web/auth/auth.go

```diff
@@ -224,9 +229,15 @@ func authorizeForm(c echo.Context) error {
 }
 
 // IsLoggedIn returns true if the context has a valid session cookie.
-func IsLoggedIn(c echo.Context) bool {
+func IsLoggedIn(c echo.Context) (bool, error) {
```

Comment:
I'm not sure why to change this. We don't care if there is an error. Maybe it's just the secret that has been rotated. We just want to know if the user is logged in, or not.